client/src/reducers/patientSubmission.js

```diff
@@ -267,11 +272,15 @@ const patientSubmissionReducer = (
         };
         draft.clinicalImpression = { ...draft.clinicalImpression, id: clinicalImpression.id };
 
+        const clinicalCgh = get(observations, `cgh[${index}]`, undefined);
+        const clinicalIndic = get(observations, `indic[${index}]`, undefined);
+        const clinicalInves = get(observations, `inves[${index}]`, undefined);
+
         draft.observations.hpos = hpos;
         draft.observations.fmh = fmhs;
-        draft.observations.cgh = { ...observations.cgh };
-        draft.observations.indic = { ...observations.indic };
-        draft.observations.summary = { ...observations.inves };
+        draft.observations.cgh = { ...clinicalCgh };
+        draft.observations.indic = { ...clinicalIndic };
+        draft.observations.summary = { ...clinicalInves };
```

Comment:
With the changes you did to the type( https://github.com/Ferlab-Ste-Justine/clin-frontend/pull/486/files#diff-5f0f95efca0c8add6b9cb68e7b3a60e0c0e7f7a99be234ca130d0b157a61ea07R37) shouldn't it be an array?

Reply:
No, it's not (see line 275 to 277)